### ISSUE TITLE
Unclear where to put additional pure-Python modules to include

### ISSUE THREAD

**Issue by barawn** [eGenix/egenix-pyrun#14]
The documentation for PyRun suggests that adding modules is just

```
The easiest way to have modules or whole packages added to PyRun is to
modify PyRun/Runtime/pyrun_extras.py and import them in that file.
```

However it's not clear where those Python modules need to be located? I edited makepyrun to print the LIBDIR directory and it looks like it's searching something like ``build/3.9-ucs4/python-installation/lib/python3.9/``, but that directory isn't there pre-build and is cleared out during ``clean`` anyway?

I made it work by making ``clean``, then ``interpreter``, then copying the modules into the above directory before making ``runtime``, but that took a lot of effort to figure out if that's the intended method.

**Comment by malemburg** [comment]
True; those details changed a lot after I refactored the project layout to support compiling multiple versions from a single source.

I should probably make it possible to point the build process to a directory where to pick up additional files and then have these copied into the build tree at build time.

Please note that the documentation on https://www.egenix.com/products/python/PyRun/ is not up to date. The README of the project is currently the most up to date version we have. I want to refactor and update the docs into Markdown and then use MkDocs to render them, but this is still in the planning stage.

**Comment by malemburg** [comment]
Thanks. I'll have a look.
